### PR TITLE
[MD] Changing "sock" declaration to "conn" to avoid "sock is not defined"

### PR DESCRIPTION
**Pull request by Dakauann**
…ndefined declaration".

just changed the "sock" declaration to "conn" on events and on reconnect mode, If u test the old code you'll receive a "sock is not defined".

### CODE REVIEW

**Comment by azudindaem** [comment]
Nothing to do with changing the name will solve the problem. Its just declaration. You can change on your self and no need to pr

**Comment by Dakauann** [comment]
> Nothing to do with changing the name will solve the problem. Its just declaration. You can change on your self and no need to pr

it is meaningless to put something undeclared in an example.

File: README.md [hunk 1]
```diff
@@ -44,30 +44,34 @@ TODO
 import makeWASocket from '@adiwajshing/baileys-md'
 
 async function connectToWhatsApp () {
+    // making a connection
     const conn = makeWASocket({
```
Comment (azudindaem) [inline README.md:48]: just change this to
const sock = makeWASocket({

**Review by azudindaem — CHANGES_REQUESTED** [review]

**Comment by azudindaem** [comment]
[Working example](https://github.com/adiwajshing/Baileys/tree/multi-device/Example)

**Comment by Dakauann** [comment]
> [Working example](https://github.com/adiwajshing/Baileys/tree/multi-device/Example)

conn is more explanatory and most used by adiwajshing

**Comment by azudindaem** [comment]
I believe the reason why adi uses sock on MD to differentiate with the legacy version. And the conn example in Readme is typo. I personally prefer sock for MD

**Pull request closed without merging**